examples/kind/konnectivity-server.yaml

```diff
@@ -53,6 +53,17 @@ spec:
       nodeSelector:
         node-role.kubernetes.io/control-plane: ""
       hostNetwork: true
+      initContainers:
+      - name: busybox
+        image: busybox
+        command:
+        - sh
+        - -c
+        - |
+          chmod -R 777 /etc/kubernetes
```

Comment:
in the manifest I see `/etc/srv/kubernetes`, should this match?

https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/master/examples/kubernetes/konnectivity-server.yaml

Reply:
> should this match?

Make sense for me.